### DISCUSSION TITLE
Quand on sait pas coder mais qu'on a des idées

### DISCUSSION THREAD

**Discussion by Jake-Freaks** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#57] [Q&A] [answered]
Coucou, 

Quand on sait pas coder ni utiliser github ni rien. Est-ce qu'on peut proposer des idées qui peuvent être travaillées par d'autres?
Comment les proposer? Je fais un post idée? Je passe par le discord? 

Merci de vos réponses.
Jake

**Comment by spacevx** [comment]
Salut,

Alors je suis pas du tout sur mais quand tu créer une <a href="https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/discussions/new/choose">discussion</a> on te propose de "partager des nouvelles idées de fonctionnalités" , j'imagine que c'est fait pour ça  
![image](https://user-images.githubusercontent.com/40030799/212761945-29ac4174-8565-4961-b107-a5dbfb16d94f.png)

**Comment by NariieL** [comment, marked as answer]
Coucou ! Pour être totalement transparente, ça ne sert pas à grand chose @Jake-Freaks. Des idées, nous en avons des tonnes. Le Github et l'Opensource sert aux devs avant tout, pas à donner des idées. Merci tout de même. <3
